test/integration/server/schema/resolvers.ts

```diff
@@ -1,10 +1,10 @@
-import { PolarisGraphQLContext } from '@enigmatis/polaris-common';
-import { PubSub } from 'apollo-server-express';
-import { DeleteResult, getPolarisConnectionManager, Like } from '../../../../src/index';
-import { TestContext } from '../context/test-context';
-import { Author } from '../dal/entities/author';
-import { Book } from '../dal/entities/book';
-import { polarisGraphQLLogger } from '../utils/logger';
+import {PolarisGraphQLContext} from '@enigmatis/polaris-common';
```

Comment:
Not prettified, execute `npm run lint:fix` and push the prettified

Reply:
OK PARTUSHKA